build/debian-base/Makefile

```diff
@@ -33,22 +33,22 @@ SUDO=$(if $(filter 0,$(shell id -u)),,sudo)
 export DOCKER_CLI_EXPERIMENTAL := enabled
 
 ifeq ($(ARCH),amd64)
-	BASEIMAGE?=debian:stretch
+	BASEIMAGE?=debian:buster
```

Comment:
seems there is a slim flavour of buster in the docker registry with half of the size `buster-slim25.84 MB` vs `buster48.05 MB` https://hub.docker.com/_/debian/?tab=tags&page=1&name=buster
is it worth to try it?

Reply:
Possibly... This is sort of the same issue as above with the `apt-get purge`. People can submit follow-up commits to slim things down further if we don't get it perfect here.

Reply:
yes.  I don't know who did the debian-base work before, but we probably need a community ownership and docs on how to repeat the process for new debian releases

Reply:
@tallclair was it you?

Reply:
@tallclair ?

Reply:
Looks like the slim version is mostly redundant with the directories we remove, but I think moving to that base can't hurt.

I agree we need a better process for this base image. I'm tempted to say that we should get out of the image maintenance game and just scrap this for `buster-slim`, but I worry about how much noise we'll end up with from the vulnerability scanners.

That would be somewhat mitigated if we do a full rebuild from the latest upstream base on every release. E.g. rebuild the intermediate images (debian-iptables) as part of the release process.

Reply:
sorry if this is orthogonal of if this was previously discussed but, based on the image size and security concerns, why using debian instead of alpine?, per example https://github.com/kubernetes/kubernetes/pull/84420